src/main/kotlin/no/nav/tilleggsstonader/sak/hendelser/personhendelse/PersonhendelseKafkaListener.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.tilleggsstonader.sak.hendelser.personhendelse
+
+import no.nav.person.pdl.leesah.Endringstype
+import no.nav.person.pdl.leesah.Personhendelse
+import no.nav.tilleggsstonader.sak.hendelser.personhendelse.dødsfall.DødsfallHendelse
+import no.nav.tilleggsstonader.sak.hendelser.personhendelse.dødsfall.DødsfallHåndterer
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.springframework.context.annotation.Profile
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.support.Acknowledgment
+import org.springframework.stereotype.Service
+
+@Service
+@Profile("!local & !integrasjonstest & !prod")
+class PersonhendelseKafkaListener(
+    private val dødsfallHåndterer: DødsfallHåndterer,
+) {
+    @KafkaListener(
+        id = "tilleggsstonader-sak",
+        topics = ["\${topics.leesah}"],
+        containerFactory = "personhendelserListenerContainerFactory",
+    )
+    fun listen(
+        consumerRecords: List<ConsumerRecord<String, Personhendelse>>,
+        acknowledgment: Acknowledgment,
+    ) {
+        consumerRecords
+            .map { it.value() }
+            .filter { it.erDødsfall() }
+            .map { it.tilDødsfallDomene() }
+            .forEach {
+                dødsfallHåndterer.håndter(it)
```

Comment:
En del av logginnslagen i den andre filen er litt vanskelige å koble til en gitt hendelseId. Vi kan ev legge til den informasjonen i MDC for hver håndter.
Då får vi med den informasjonen for alle logger som et eget felt i logginslagen. 

```kotlin
try {
                    MDC.put("hendelseId", it.hendelseId.toString())
                    dødsfallHåndterer.håndter(it)
                } finally {
                    MDC.remove("hendelseId")
                }
```